### PR TITLE
GSREN3D-327 - Home page and line page: Labeling POI while Hoovering

### PR DESCRIPTION
**Pull request by toni-garcia-camptocamp**
GSREN3D-327 - Home page and line page: Labeling POI while Hoovering

### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-327

### Description

https://user-images.githubusercontent.com/121813008/220166927-db80f987-c6ca-410b-ba0a-731751ff4985.mp4

### CODE REVIEW

File: src/services/poi.ts [hunk 1]
```diff
@@ -29,3 +34,34 @@ export async function fixGeometryOfPoi(rennesApp: RennesApp) {
...
+  const styleItem = generatePoiStyle(
+    feature.getProperties()['site_nom'],
+    feature.getProperties()['distance'],
+    map3dStore.is3D(),
+    false
+  )
+  feature.setStyle(styleItem.style)
+}
+
+export function undisplayPoiExpectCurrent() {
```
Comment (ismailsunni) [inline src/services/poi.ts:50]: Could you add some comments about this function? The name is getting quite "strange" :)

**Review by ismailsunni — APPROVED** [review]
Great!

**Comment by toni-garcia-camptocamp** [comment]
> 

I have add a line break when the name is more than 25 characters (the CHAR_MAX const)
image

**Comment by toni-garcia-camptocamp** [comment]
> Nice! Seems fluid on the video. Don't we want also a line break with long words? Maybe the icon could be bigger when hovering? I don't find example on figma

There are not example on figma, the demand is describe in the ticket: https://jira.camptocamp.com/browse/GSREN3D-327

**Pull request merged**